luma/led_matrix/device.py

```diff
@@ -59,9 +60,27 @@ def __init__(self, serial_interface=None, width=8, height=8, cascaded=None, rota
             raise luma.core.error.DeviceDisplayModeError(
                 "Unsupported display mode: {0} x {1}".format(width, height))
 
+        assert block_orientation in [0, 90, -90, "horizontal", "vertical"]
+        if block_orientation == "vertical":
+            msg = (
+                "WARNING! block_orientation=\"vertical\" is now deprecated and "
+                "should be changed to block_orientation=-90 to acheive the same "
+                "effect. Use of \"vertical\" will be removed entirely beginning "
```

Comment:
was just being consistent with other deprecation version with 'v' in version. Not too fussed about the wording here as they will be gone soon.